html2text/__init__.py

```diff
@@ -376,15 +381,38 @@ def handle_tag(self, tag, attrs, start):
                 self.blockquote -= 1
                 self.p()
 
+        def no_preceding_space(self):
```

Comment:
Instead of using an if condition here, could this be on-line like:
```
def no_preceding_space(self):
     return self.preceding_data and re.match(r'[^\s]', self.preceding_data[-1])
```

In this way the function always returns a `bool`.